docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -79,14 +79,48 @@ function AppMenuExtensionPoint() {
 }
 ```
 
-#### Why does the extension have `onClick` and `path`?
+### Why does the extension have `onClick` and `path`?
 
 Each extension link has either a `path` or an `onClick` property defined. There's never a scenario where both properties are defined at the same time.
 
 The reason for this behavior is that we want to be able to support both native browser links and callbacks. If the plugin adding the extension wants to navigate the user away from the current view into their app, then they can choose to define a path.
 
 If instead you want to open a modal or trigger a background task without sending the user away from the current page, then you can provide a callback.
 
+### Example: create an extension point for displaying components
+
+Component type extensions are simply React components, which you can render anywere in your plugin.
+
+```tsx title="src/components/Toolbar.tsx"
+import { getPluginComponentExtensions } from '@grafana/runtime';
+
+export const Toolbar = () => {
+  const { extensions } = getPluginComponentExtensions({ extensionPointId: '<extension-point-id>' });
+  const version = '1.0.0'; // Let's share this with the extensions
```

Comment:
it might be good to describe the intent in this example in the introduction text as well as code comments

Reply:
I have tried to make it more clear in https://github.com/grafana/plugin-tools/pull/548/commits/a828ad4227e129281442655b3296a389642e6581.